mirgecom/fluid.py

```diff
@@ -230,7 +232,7 @@ class ConservedVars:
     @property
     def array_context(self):
         """Return an array context for the :class:`ConservedVars` object."""
-        return self.mass.array_context
+        return get_container_context_recursively(self.mass)
```

Comment:
This seems to throw some warnings with an upcoming error in J2022:
```
test/test_symbolic.py: 2 warnings
  /Users/lukeo/repos/mirgecom-review/emirge/mirgecom/mirgecom/fluid.py:235: DeprecationWarning: No array context was found. This will be an error starting in July of 2022. If you would like the function to return None if no array context was found, use get_container_context_recursively_opt.
    return get_container_context_recursively(self.mass)
```

Reply:
https://github.com/inducer/arraycontext/blob/main/arraycontext/container/__init__.py#L301

Reply:
This is very disconcerting.   I had fixed this already.  How am I losing these changes?

Reply:
Oh n/m i see this was not a suggestion, but a highlight of a change I made.   This change was _supposed_ to address that deprecation.

Reply:
#672